design/ceph/object/store.md

```diff
@@ -70,6 +72,11 @@ If `preservePoolsOnDelete` is set to 'true' the pools used to support the object
     erasureCoded:
       dataChunks: 6
       codingChunks: 2
+  radosNamespaces:
+    # If true, create RADOS namespaces for this object store.
+    # Default is false, preserve the current behavior for backward compatibility.
+    create: true
+    poolName: my-pool-collection
```

Comment:
Is there a case where a user might want to use different pools for metadata and data?

Reply:
In the example from my understanding only one pool `collection` mentioned and used for both data and metadata pools. I guess we can have seperate pool  for data and metadata as well. @cbodley any ideas?